### PR TITLE
#119: added example using the new caret_position variable

### PR DESCRIPTION
**Pull request by vrajur**
# Description 
Added an example to the docs using the new `caret_position` variable. This is associated with [issue #119](https://github.com/Taitava/obsidian-shellcommands/issues/119) in the obisidian-shellcommands repo

Change Type: enhancement

### CODE REVIEW

File: Example shell commands/Open the current file in vscode and jump to the current position.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Example
+
+## Windows/Mac/Linux
+ - If you have the [Visual Studio Code] editor installed on your system (and the shell command available), you can easily jump between obsidian and VSCode using the following SC command: `code --goto {{file_position:absolute}}:{{caret_position}}`
```
Comment (Taitava) [inline Example shell commands/Open the current file in vscode and jump to the current position.md]: - I think VSCode link is missing
- file_position -> file_path
- obsidian -> capital O

```suggestion
 - If you have the [Visual Studio Code](https://code.visualstudio.com/) editor installed on your system (and the shell command available), you can easily jump between Obsidian and VSCode using the following SC command: `code --goto {{file_path:absolute}}:{{caret_position}}`
```

**Review by Taitava — CHANGES_REQUESTED** [review]
This is an excellent example! 😎 I didn't realize before how useful the caret position is when opening a file in another application! Cheers! 🥂

File: Example shell commands/Open the current file in vscode and jump to the current position.md (hunk 1 above)
Comment (vrajur) [reply Example shell commands/Open the current file in vscode and jump to the current position.md]: Ahh that was sloppy, I wrote it up very quickly (too quickly it seems haha). Thanks for catching that!

**Comment by vrajur** [comment]
Thanks for the review! Yeah this example was the initial reason I wanted to add the variable (so I could jump between vscode and obsidian when editing) - it works great! I'll have to checkout some the other suggestions in the discussion from [#114](https://github.com/Taitava/obsidian-shellcommands/discussions/114) as well. Thanks for creating this super handy plugin :slightly_smiling_face:

**Pull request merged**